### PR TITLE
Add main computer Tab

### PR DESCRIPTION
**Pull request by iamsolankiamit**
Change-type: minor
Signed-off-by: Amit Solanki <amit@balena.io>

### CODE REVIEW

**Comment by iamsolankiamit** [comment]
@balena-ci rebase

File: src/components/GetStarted.tsx [hunk 1]
```diff
@@ -229,99 +238,237 @@ const GetStarted = ({
...
-							pt={2}
-						>
-							<LazyImage src={tasksImg} alt="Rosetta tasks on your device" />
-						</Flex>
-					</Step>
+					{isMainComputer ? (
+						<>
+							<Step index={1}>
+								<Txt fontSize={2} mb={2} bold>
+									Download and Install BOINC
```
Comment (rahul-thakoor) [inline src/components/GetStarted.tsx:283]: can we link to appropriate installer based on user agent as we do for etcher?

Comment (iamsolankiamit) [reply src/components/GetStarted.tsx:283]: Yes, I'll work on it next, wanted to get something out before doing that, so  it can be reviewed.

Comment (rahul-thakoor) [reply src/components/GetStarted.tsx:283]: awesome! thanks. I agree it is not high priority

**Comment by iamsolankiamit** [comment]
@balena-ci rebase

**Review by sradevski — APPROVED** [review]

File: src/components/LazyImage.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ export const LazyImage = (props: { src: string; alt: string }) => {
 
 		observerRef.current.observe(imageRef.current);
 
-		return () => observerRef.current.disconnect();
+		return () => observerRef.current && observerRef.current.disconnect();
```
Comment (sradevski) [inline src/components/LazyImage.tsx:26]: I was going to say that lazy loading didn't work :)

**Pull request merged**